testplan/runners/pools/remote.py

```diff
@@ -468,62 +525,79 @@ def _prepare_remote(self):
-                    exclude=self.cfg.pull_exclude)
+                    exclude=self.cfg.pull_exclude,
+                )
 
     def _fetch_results(self):
         """Fetch back to local host the results generated remotely."""
-        self.logger.debug('Fetch results stage - %s', self.cfg.remote_host)
+        self.logger.debug("Fetch results stage - %s", self.cfg.remote_host)
         try:
             self._transfer_data(
                 source=self._remote_testplan_runpath,
                 remote_source=True,
-                target=self.parent.runpath)
+                target=self.parent.runpath,
+            )
             if self.cfg.pull:
                 self._pull_files()
         except Exception as exc:
             self.logger.exception(
-                'While fetching result from worker [%s]: %s', self, exc)
+                "While fetching result from worker [%s]: %s", self, exc
+            )
 
     def _proc_cmd(self):
         """Command to start child process."""
-        if platform.system() == 'Windows':
-            if platform.python_version().startswith('3'):
-                python_binary = os.environ['PYTHON3_REMOTE_BINARY']
+        if platform.system() == "Windows":
+            if platform.python_version().startswith("3"):
+                python_binary = os.environ["PYTHON3_REMOTE_BINARY"]
             else:
-                python_binary = os.environ['PYTHON2_REMOTE_BINARY']
+                python_binary = os.environ["PYTHON2_REMOTE_BINARY"]
         else:
             python_binary = sys.executable
 
-        cmd = [python_binary, '-uB',
-               self._child_paths.remote,
-               '--index', str(self.cfg.index),
-               '--address', self.transport.address,
-               '--type', 'remote_worker',
-               '--log-level', str(TESTPLAN_LOGGER.getEffectiveLevel()),
-               '--wd', self._working_dirs.remote,
-               '--runpath', self._remote_testplan_runpath,
-               '--remote-pool-type', self.cfg.pool_type,
-               '--remote-pool-size', str(self.cfg.workers),
-               '--testplan', self._testplan_import_path.remote,
-               '--sys-path-file', self._write_syspath()]
+        cmd = [
+            python_binary,
+            "-uB",
+            self._child_paths.remote,
+            "--index",
+            str(self.cfg.index),
+            "--address",
+            self.transport.address,
+            "--type",
+            "remote_worker",
+            "--log-level",
+            str(TESTPLAN_LOGGER.getEffectiveLevel()),
+            "--wd",
+            self._working_dirs.remote,
+            "--runpath",
+            self._remote_testplan_runpath,
+            "--remote-pool-type",
+            self.cfg.pool_type,
+            "--remote-pool-size",
+            str(self.cfg.workers),
+            "--testplan",
+            self._testplan_import_path.remote,
+            "--sys-path-file",
+            self._write_syspath(),
+        ]
```

Comment:
Unfortunate that we'll lose these types of formattings, i.e. the unformatted bit on the left is more readable in this case. THough overall its a fine tradeoff for consistent style throughout.